tests/jenkins/TestSignArtifacts.groovy

```diff
@@ -57,6 +64,48 @@ class TestSignArtifacts extends BuildPipelineTest {
             return helper.callClosure(closure)
         })
 
-        super.testPipeline("jenkins/sign-artifacts/sign-standalone-artifacts.jenkinsfile", "tests/jenkins/jenkinsjob-regression-files/sign-standalone-artifacts.jenkinsfile")
+        super.testPipeline("jenkins/sign-artifacts/sign-standalone-artifacts.jenkinsfile",
+                "tests/jenkins/jenkinsjob-regression-files/sign-standalone-artifacts/sign-standalone-artifacts.jenkinsfile")
+
+        verifySignArtifactsParams(helper, '.sig', 'linux', 'workspace/artifacts/')
+
+
+        def filenamesForUrls = ['dummy_1_artifact.tar.gz', 'dummy_1_artifact.tar.gz.sig',
+                                'dummy_2_artifact.tar.gz', 'dummy_2_artifact.tar.gz.sig']
+        TestPrintArtifactDownloadUrlsForStaging.verifyPrintArtifactDownloadUrlsForStagingParams(helper, filenamesForUrls,
+                'sign_artifacts_job/dummy/upload/path/20/dist/signed')
+    }
+
+    static void verifySignArtifactsParams(helper, signatureType, distributionPlatform, artifactPath) {
```

Comment:
Thanks for adding a verify helper for signing. It is good that both the `TestSignArtifacts` and `TestDataPrepperDistributionArtifacts` class share the same verification code. 

But, I think it might make sense to move this into its own class. It is uncommon for one test to call methods on another test. I think it is confusing to new audiences. Perhaps just move this into a `SignArtifactsTestHelper` class.

Reply:
The issue with that is that we'll endup having one class associated with every test class holding one function. Maybe we can keep the `verifyParamsForSignArtifacts` in the `signArtifacts` test itself, since this is essentially testing for `signArtifacts`